inst/include/code.h

```diff
@@ -0,0 +1,8 @@
+#ifndef FROGGER_H
```

Comment:
If you want to simplify this, you can (epirically at least) use `#pragma once`. See mode for examples (e.g., https://github.com/mrc-ide/mode/blob/main/inst/include/mode/control.hpp)